colin-api/src/colin_api/models/filing.py

```diff
@@ -1395,13 +1396,18 @@ def add_filing(cls, con, filing: Filing) -> int:
                                                 Business.TypeCodes.BCOMP.value,
                                                 Business.TypeCodes.BCOMP_CONTINUE_IN.value,
                                             ])
+            is_business_in_lear = cls.is_business_in_lear(lear_identifier)
 
-            # Freeze all entities except CP if 'enable-bc-ccc-ulc' flag is on else just freeze BEN
+            # Freeze all entities except CP if business exists in lear and
+            # 'enable-bc-ccc-ulc' flag is on else just freeze BEN
             is_frozen_condition = (
                 flags.is_on('enable-bc-ccc-ulc') and
-                business['business']['legalType'] != Business.TypeCodes.COOP.value
+                business['business']['legalType'] != Business.TypeCodes.COOP.value and
+                is_business_in_lear
             )
 
+            current_app.logger.debug(f'Business {lear_identifier}, is_frozen_condition:{is_frozen_condition}')
```

Comment:
so since you end up doing the `cls.is_business_in_lear(lear_identifier)` call upfront, can you add another debug logging statement right after `is_business_in_lear = cls.is_business_in_lear(lear_identifier)`?